docs/conversation.md

```diff
@@ -426,3 +426,26 @@ Get all (for moderators and in case of "free selection") or the assigned breakou
         + `400 Bad Request` When the conversation is a breakout room
         + `403 Forbidden` When the current user is not a moderator/owner or the conversation is not a public conversation
         + `404 Not Found` When the conversation could not be found for the participant
+
+# Allow non moderators to mention everyone using `@all`
+* Required capability: `canMentionEveryone`
+* Method: `POST`
+* Endpoint: `/room/{token}/can-mention-everyone`
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `403 Forbidden` When the current user is not a moderator/owner or the conversation is not a public conversation
+        + `404 Not Found` When the conversation could not be found for the participant
+
+
+# Disallow non moderators to mention everyone using `@all`
+* Required capability: `canMentionEveryone`
+* Method: `DELETE`
+* Endpoint: `/room/{token}/can-mention-everyone`
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `403 Forbidden` When the current user is not a moderator/owner or the conversation is not a public conversation
```

Comment:
Should that be group conversation? Or is it really limited to public?

Reply:
```suggestion
        + `403 Forbidden` When the current user is not a moderator/owner or the conversation is a one-to-one conversation
```

Reply:
But actually 1-1 should return a 400